src/main/resources/db/migration/common/V1_228__add_unique_to_transcription.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE transcription
```

Comment:
The DBA team will need to be made aware of this CONSTRAINT so that it doesn't get lost later on
if/when we dump our developer migrations done so far - `modernised_darts_schema.sql`